src/main/java/seedu/address/model/displayable/Person.java

```diff
@@ -34,6 +35,7 @@ public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tag
         this.email = email;
         this.address = address;
         this.tags.addAll(tags);
+        this.priority = new Priority("medium");
```

Comment:
Maybe add an extra type of priority such as "blank", such that when priority is blank it will not display on the UI.

Reply:
Done in the latest commit